src/schemes/fluid/viscosity.jl

```diff
@@ -4,20 +4,7 @@ function dv_viscosity(particle_system, neighbor_system,
                       v_particle_system, v_neighbor_system,
                       particle, neighbor, pos_diff, distance,
                       sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-    viscosity = viscosity_model(neighbor_system)
-
-    return dv_viscosity(viscosity, particle_system, neighbor_system,
-                        v_particle_system, v_neighbor_system,
-                        particle, neighbor, pos_diff, distance,
-                        sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-end
-
-function dv_viscosity(particle_system, neighbor_system::OpenBoundarySPHSystem,
-                      v_particle_system, v_neighbor_system,
-                      particle, neighbor, pos_diff, distance,
-                      sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-    # No viscosity in the open boundary system. Use viscosity of the fluid system.
-    viscosity = viscosity_model(particle_system)
+    viscosity = viscosity_model(neighbor_system, particle_system)
```

Comment:
Why is this using the viscosity model of the neighbor system? Can you please add a comment?

Reply:
I don't know this was like this before. So changing that again changes the validation files again :D.

Reply:
I think because we need to use the boundary viscosity for fluid-boundary interaction, for example.
I guess it would then make sense to change this to `viscosity_model(particle_system, neighbor_system)` and let `viscosity_model` return the viscosity of the `neighbor_system` by default, except when this is an open boundary system.